lib/postgrex/protocol.ex

```diff
@@ -964,6 +1021,8 @@ defmodule Postgrex.Protocol do
     do: {:commit, nil}
   defp decode_tag("ROLLBACK"),
     do: {:rollback, nil}
+  defp decode_tag( :suspended ),
+    do: {:select, nil}
```

Comment:
Ah ok, I misunderstood what was going on :D.

We can't use `:select` as I believe an insert or update with returning could be used here. Lets use `{:stream, nil}`.